types/index.d.ts

```diff
@@ -14,28 +14,28 @@ declare module 'ember-cli-page-object' {
   function collection<T extends Partial<Definition>>(scope: string, definition?: T): Collection<T>;
 
   // Attributes
-  function attribute(attributeName: string, scope?: string): GetterDescriptor<string>;
-  function isVisible(scope?: string): GetterDescriptor<boolean>;
-  function isHidden(scope?: string): GetterDescriptor<boolean>;
-  function isPresent(scope?: string): GetterDescriptor<boolean>;
-  function text(scope?: string): GetterDescriptor<string>;
-  function value(scope?: string): GetterDescriptor<string>;
-  function property(name: string): GetterDescriptor<any>;
-  function property(scope: string, name: string): GetterDescriptor<any>;
-  function hasClass(className: string): GetterDescriptor<boolean>;
-  function hasClass(scope: string, className: string): GetterDescriptor<boolean>;
-  function notHasClass(className: string): GetterDescriptor<boolean>;
-  function notHasClass(scope: string, className: string): GetterDescriptor<boolean>;
-  function contains(scope?: string): (text: string) => GetterDescriptor<boolean>;
-  function count(scope?: string): () => GetterDescriptor<boolean>;
+  function attribute(attributeName: string, scope?: string, options?: FindOptions): GetterDescriptor<string>;
+  function isVisible(scope?: string, options?: FindOptions): GetterDescriptor<boolean>;
+  function isHidden(scope?: string, options?: FindOptions): GetterDescriptor<boolean>;
+  function isPresent(scope?: string, options?: FindOptions): GetterDescriptor<boolean>;
+  function text(scope?: string, options?: FindOptions): GetterDescriptor<string>;
+  function value(scope?: string, options?: FindOptions): GetterDescriptor<string>;
+  function property(name: string, options?: FindOptions): GetterDescriptor<any>;
```

Comment:
looking at the current implementation https://github.com/san650/ember-cli-page-object/blob/master/addon-test-support/properties/property.js#L69

I seems like it would not workm cause with such signatire it would call smth like:

```
findElementWithAssert(this, findOptions, {});
```
which is not expected.

I think this line should stay unchanged